GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategoryByCospans.gi

```diff
@@ -384,16 +384,13 @@ end );
 
 ##
 InstallMethod( AsGeneralizedMorphismByCospan,
-        [ IsCapFunctor, IsString, IsCapCategory, IsCapCategory ],
+        [ IsCapFunctor, IsString ],
```

Comment:
General remark on this: Those lifts of functors are now only implemented for generalized morphisms by cospans. They should be implemented for spans and three arrows, too, and then there should be general methods to lift them (i.e., as in `Switcher.gi`). This makes it possible to implement an algorithm without specifying the generalized morphism standard, so the standard can be easily switched to see which type of generalized morphism performs best.